src/broker/boot_pmi.c

```diff
@@ -88,7 +88,6 @@ static int set_broker_mapping_attr (struct upmi *upmi,
     else {
         /* First attempt to get flux.taskmap, falling back to
          * PMI_process_mapping if this key is not available.
-         * This should be replaced when #4800 is fixed.
```

Comment:
Just to clarify, the broker still does need to fall back to PMI_process_mapping here?

Reply:
Yes, the current code which does
```c
        char *s;
        if (upmi_get (upmi, "flux.taskmap", -1, &s, NULL) == 0
            || upmi_get (upmi, "PMI_process_mapping", -1, &s, NULL) == 0) {
            val = pmi_mapping_to_taskmap (s);
            free (s);
        }
```
is fine I think!  In any case, the issue was closed so the reference probably should be dropped.